api/types/access_request.go

```diff
@@ -66,6 +72,21 @@ type AccessRequest interface {
 	GetSystemAnnotations() map[string][]string
```

Comment:
General note to @russjones, @andrejtokarcik and @fspmarshall. This access/review/threshold system is very expressive. This raises the importance of:

https://github.com/gravitational/teleport/pull/4991 and https://github.com/gravitational/teleport/issues/3731

without tracing and role scenario testers, it will be extremely hard to troubleshoot and develop